cylc/uiserver/resolvers.py

```diff
@@ -316,9 +316,18 @@ async def play(cls, workflows, args, workflows_mgr, log):
 
                 if ret:
                     # command failed
-                    _, err = proc.communicate()
+                    out, err = proc.communicate()
+                    if 'previously run with' in out:
+                        # Can't upgrade:
+                        msg = out + '\nEdit the play command to use "upgrade".'
```

Comment:
I think this should be in cylc-flow at https://github.com/cylc/cylc-flow/blob/ce6b7bf34e084c754e64472e5bce64d143b28c10/cylc/flow/scheduler_cli.py#L525-L526

Reply:
The message should be stored there?